### PR TITLE
Add "rake test" command

### LINKED ISSUE #398: Rake test command
To make testing simpler, we should combine running RSpec and Rubocop into a single command, `rake test`

### PR DESCRIPTION
**Pull request by J3RN**
I added the "rake test" command, configured Travis CI to use it, and suggested that contributors use it in the contributing guidelines.

Closes #398.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -27,6 +27,4 @@ before_install:
   - ./configure && make && sudo make install
   - cd ..
   - tmux -V
-script:
-  - bundle exec rspec spec
-  - bundle exec rake rubocop
+script: bundle exec test
```
Comment (adamstrickland) [inline .travis.yml]: should this be `bundle exec rake test`?

Comment (J3RN) [inline .travis.yml]: You are 100% right.

**Comment by Soliah** [comment]
I think you can also remove the old hound tasks.

**Comment by J3RN** [comment]
@Soliah Done!

**Comment by J3RN** [comment]
I'm thinking that since this change is internal-facing, there's no need to update the CHANGELOG for it. Let me know if you think otherwise.

**Comment by adamstrickland** [comment]
:+1:

**Comment by adamstrickland** [comment]
So shall we :shipit:?

**Comment by J3RN** [comment]
@adamstrickland I feel good to!

**Pull request merged**